lib/eject.js

```diff
@@ -301,37 +280,37 @@ async function eject() {
     await cliVersion.verifyLatestVersion();
 
     if (!gitUtils.isGitClean()) {
-      throw Error('Uncommitted changes found. Please commit or stash any changes before ejecting your project.');
+      throw new Error('Uncommitted changes found. Please commit or stash any changes before ejecting your project.');
     }
 
-    const skyuxConfig = getSkyuxConfig();
-    let projectName = getEjectedProjectName(skyuxConfig);
-    const ejectedProjectPath = path.join(CWD, EJECTED_FILES_TEMP_DIR);
-
+    const ejectedProjectPath = path.join(CWD, constants.EJECTED_FILES_TEMP_DIR);
     if (fs.existsSync(ejectedProjectPath)) {
       throw new Error(`The '${ejectedProjectPath}' directory already exists. Delete the directory and rerun the "eject" command.`);
     }
 
-    const libraryPath = path.join(process.cwd(), 'src/app/public');
-    const isLibrary = fs.existsSync(libraryPath);
-
-    // Ensure SPA name and library name are not the same.
-    if (isLibrary) {
-      projectName += '-spa';
-    }
-
     const gitOriginUrl = gitUtils.getOriginUrl();
     const isInternal = gitOriginUrl.startsWith('https://blackbaud.visualstudio.com/');
 
     const strictMode = await promptForStrictMode(CWD);
 
+    const libraryPath = path.join(process.cwd(), 'src/app/public');
+    const isLibrary = fs.existsSync(libraryPath);
+    /* istanbul ignore if */
+    if (isLibrary) {
+      logger.warn('Ejecting SKY UX libraries is not currently supported. Coming soon!');
```

Comment:
This will be replaced with a `return migrateLibrary()` function after being merged (currently in another branch).